src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
+import java.util.Map;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+    private float loadFactor;
```

Comment:
those two are useless fields, please remove them, it will simplify the code

Reply:
but this two fields is a need in HashMap.....